variables.data_lake.tf

```diff
@@ -0,0 +1,45 @@
+variable "storage_data_lake_gen2_filesystem" {
+  type = object({
```

Comment:
Is this a 1-2-1, cannot specify more than 1 per storage account?

Reply:
yes. This is file System deployment